src/main/java/org/isf/examination/rest/ExaminationController.java

```diff
@@ -0,0 +1,156 @@
+package org.isf.examination.rest;
```

Comment:
is not a @DeleteMapping required fr this controller/module?

Reply:
it has been introduced in "core" after OP-6 branch, I don't know if it's better to add now or later after merge into develop, wdyt?

Reply:
I would open a linked ticket/task with 2 subtasks to keep that work in pending on track and then merge this PR in order someone could start working on the respective test suite in parallel (OP-197) 

Task: Add delete mapping to ExaminationController

Subtask 1: sync op-6 branch with change on develop

subtask 2: once subtask 1 is completed, make the addition to the controller

Reply:
OK, so I resolve this conversation concerning this PR.

About the tasks, there's no need to "sync op-6 branch" (subtask1) if we postpone the work after OP-6 is completed, because its completion means it will be already merged into develop